docs/development_suite/api-console/api-design/api_key.md

```diff
@@ -46,3 +46,10 @@ curl --request GET
 :::info
 Check out the [manage the security of your endpoints section](/development_suite/api-console/api-design/endpoints.md#manage-the-security-of-your-endpoints) to know in detail how to write the user group expression
 :::
+
+:::warning
```

Comment:
i'd use a tip with a more positive message, this is a security measure after all :)

Reply:
Done 😃